### PR TITLE
fix locale and link issues with contributors_extra

### LINKED ISSUE #1882: contributors_extra_url is not rendering link correctly
The URL doesn't click. This is what is rendered in dev tools:

```html
<p class="disclaimer" data-l10n-id="trackingprotectionContributorsextra" target="_blank" href="https://mzl.la/2fYneDq">
This experiment is based on Firefox Tracking Protection technology built by Mozilla employees and contributors over the past several years.
Learn more about <a>Firefox Tracking Protection</a>.
</p>
```

Ref: [the commit](https://github.com/mozilla/testpilot/commit/1e394f2426347f63673a26a94e4142e8b6ac785e) that this was added in.

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #1882

### CODE REVIEW

File: locales/en-US/app.ftl [hunk 1]
```diff
@@ -144,6 +144,7 @@ enableExperimentTransition = Enabling...
 measurements = Your privacy
 experimentPrivacyNotice = You can learn more about the data collection for {$title} here.
 contributorsHeading = Brought to you by
+contributorsExtraURL = Learn more
```
Comment (chuckharmston) [inline locales/en-US/app.ftl]: Could we name this something more descriptive for localizers? `contributorsExtraLearnMore` maybe?

File: locales/en-US/experiments.ftl [hunk 2]
```diff
@@ -96,4 +97,5 @@ universalsearchToursteps2Copy = <p>You can always give us feedback or disable Un
 universalsearchContributors0Title = Product Manager
 universalsearchContributors1Title = Senior UX Designer
 universalsearchContributors2Title = Staff Engineer
-universalsearchContributors3Title = Senior Engineer
\ No newline at end of file
+universalsearchContributors3Title = Senior Engineer
+universalsearchEolwarning = We will automatically disable the Universal Search experiment and report the results when it ends.
```
Comment (chuckharmston) [inline locales/en-US/experiments.ftl:101]: Oops.

File: frontend/src/app/containers/ExperimentPage.js [hunk 3]
```diff
@@ -367,11 +367,11 @@ export class ExperimentDetail extends React.Component {
...
                         ))}
                       </ul>
-                    {contributors_extra && <p className="disclaimer"
-                        dangerouslySetInnerHTML={createMarkup(contributors_extra)}
-                        data-l10n-id={this.l10nId('contributors_extra')}
-                        target="_blank"
-                        href={contributors_extra_url} />}
+                    {contributors_extra && <p className="disclaimer">
+                        <span data-l10n-id={this.l10nId('contributors_extra')}>{contributors_extra}</span>&nbsp;
+                        <a data-l10n-id="contributorsExtraURL" href={contributors_extra_url} target="_blank">Learn more</a>.
```
Comment (chuckharmston) [inline frontend/src/app/containers/ExperimentPage.js]: Is `contributors_extra_url` required? If so, we should call that out explicitly in docs. If not, it should be in a conditional here.

**Review by chuckharmston — APPROVED** [review]
Few nits, but it looks good. Feel free to self-merge at your discretion.

File: locales/en-US/app.ftl (hunk 1 above)
Comment (johngruen) [reply locales/en-US/app.ftl]: Sure

File: locales/en-US/experiments.ftl (hunk 2 above)
Comment (johngruen) [reply locales/en-US/experiments.ftl:101]: 🙄

**Comment by johngruen** [comment]
@chuckharmston, fixed nits...maybe give the `ExperimentPage` container one more glance and merge?

**Review by chuckharmston — APPROVED** [review]

**Pull request merged**